docs/_documentations/codewind-openapi-eclipse.md

```diff
@@ -18,6 +18,16 @@ The Codewind OpenAPI Tools for Eclipse includes wizards that invoke the OpenAPI
 2. [Optional] Install [Codewind from the Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind).
 3. Install the [Codewind OpenAPI Tools from the Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind).
 
+## Generating and building client and server stubs by using the OpenAPI tools
+1. Right-click your Codewind project from the Codewind view and select either the **Generate Server from OpenAPI Definition** menu or **Generate Client from OpenAPI Definition** action. You are then prompted to provide options to the OpenAPI generator.
+2. Select your OpenAPI document file. The file must be a valid OpenAPI 3.0 `.yaml` or `.json` file that is in the project.
+3. Select the client or server generator type.
+4. When you're prompted to select the output folder, keep the default selection. The project root, which is the parent of the source folder, is selected by default.
+5. Click **Select As Output Folder**.
+6. When prompted with the warning message, click **Yes** to merge the generated `.pom` file with the existing `.pom` file. The existing `pom.xml` file is backed up as `pom-backup.xml` or as a variation of this name. The new `pom.xml` file is a merged version of the original `.pom` file and the `.pom` file that is generated. The merge attempts to include all necessary dependencies for the project to build properly. Because of limitations in the merge process, XML comments, such as `<!-- example comment -->`, from the original `.pom` file are removed. For any Codewind project, if autobuild is turned on, the project builds automatically.
+7. See that your `src` folder shows the newly generated code. Depending on the generator type that you chose, a `gen` package is created. Your Codewind project state returns to the `[Running]` state. The target source folders, `src/main/java` and `src/test/java`, are not customizable.
+8. Now, you can customize the application logic.
+
```

Comment:
@sishida , the Eclipse steps are different, so we can't use the steps from VS Code here.  Steps 6 to 8 above however are common enough to repeat in the Eclipse help.  Those steos should be appended to step 5 or 6 under the Running commands section below.

S

Reply:
Under the `Running commands` section, step 4, `Select an action to generate the client or server stubs or HTML documentation` essentially is the start of this _new_ section entitled `Generating and building client and server stubs by using the OpenAPI tools`

Reply:
For example, something like this:
4. Select an action to generate the client or server stubs or HTML documentation.
5. If client or server stub is selected,
6. Enter the requested information that is displayed in the wizard.
- For output folder, keep the default selection. The project root, which is the parent of the source folder, is selected by default.
7. Click Finish
8. When prompted with the warning message, click **Yes** to merge the generated `.pom` file with the existing `.pom` file. The existing `pom.xml` file is backed up as `pom-backup.xml` or as a variation of this name. The new `pom.xml` file is a merged version of the original `.pom` file and the `.pom` file that is generated. The merge attempts to include all necessary dependencies for the project to build properly. Because of limitations in the merge process, XML comments, such as `<!-- example comment -->`, from the original `.pom` file are removed. For any Codewind project, if autobuild is turned on, the project builds automatically.
9. See that your `src` folder shows the newly generated code. Depending on the generator type that you chose, a `gen` package is created. Your Codewind project state returns to the `[Running]` state. The target source folders, `src/main/java` and `src/test/java`, are not customizable.
10. Now, you can customize the application logic.
11. After generation, edit the .openapi-generator-ignore file to ensure that subsequent code generation does not overwrite custom code.


After generation, edit the .openapi-generator-ignore file to ensure that subsequent code generation does not overwrite custom code.